plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineWizardPage.java

```diff
@@ -64,7 +67,9 @@ public void createControl(Composite parent) {
     createCustomFields(container, pageValidator);
 
     // Manage APIs
-    appEngineLibrariesSelectorGroup = new AppEngineLibrariesSelectorGroup(container);
+    if (showLibrariesSelectorGroup) {
```

Comment:
Rather than an if block here I would expect this to be managed via the subclasses. This code doesn't need to be present in the superclass since it doesn't apply to all subclasses.